velox/exec/Spill.h

```diff
@@ -456,6 +456,9 @@ class TestScopedSpillInjection {
 /// Test utility that returns true if triggered spill is evaluated to happen,
 /// false otherwise.
 bool testingTriggerSpill();
+
+/// Removes empty partitions from given spill partition set.
+void removeEmptyPartitions(SpillPartitionSet& partitionSet);
```

Comment:
This is a nice refactoring. Would it be possible to extract it into a separate PR to make this PR smaller and a bit easier to follow?

Reply:
Got it, Done. https://github.com/facebookincubator/velox/pull/9091